### PR TITLE
Add course language

### LINKED ISSUE #512: Course Language
Since QGIS is a multinational, multilingual effort, the certification program would allow courses in other languages besides English. 

course

### PR DESCRIPTION
**Pull request by sonlinux**
Fix #512 
## On course list page
course_list

## On course detail page
course_detail

## Passing updated tests for course model.
passing_tests_for_language

### CODE REVIEW

File: django_project/certification/models/course.py [hunk 1]
```diff
@@ -32,6 +32,15 @@ class Course(models.Model):
         blank=True,
     )
 
+    course_language = models.CharField(
+        help_text=_(
+            'A language that the Course will be conducted in,  e.g. English.'),
```
Comment (ismailsunni) [inline django_project/certification/models/course.py]: double space?

File: django_project/certification/models/course.py [hunk 2]
```diff
@@ -32,6 +32,15 @@ class Course(models.Model):
         blank=True,
     )
 
+    course_language = models.CharField(
```
Comment (ismailsunni) [inline django_project/certification/models/course.py]: Would make it redundant to name it course_language?

**Comment by sonlinux** [comment]
Thanks @ismailsunni 
I have updated the code.

File: django_project/certification/forms.py [hunk 3]
```diff
@@ -272,6 +272,7 @@ class Meta:
             'course_type',
             'course_convener',
             'training_center',
+            "language",
```
Comment (ismailsunni) [inline django_project/certification/forms.py]: single quote

File: django_project/certification/templates/course/detail.html [hunk 4]
```diff
@@ -75,6 +75,7 @@ <h1>{{ course.course_type.name }} ({{ course.start_date }} to {{ course.end_date
             <div class="details col-lg-10" style="width: 60%">
                 <span class="grey-italic col-lg-3">Course name</span><span class="col-lg-9">{{ course.course_type.name }} ({{ course.start_date }} to {{ course.end_date }})<br/></span>
                 <span class="grey-italic col-lg-3">Course type</span><span class="col-lg-9">{{ course.course_type }}<br/></span>
+                <span class="grey-italic col-lg-3">Course Language</span><span class="col-lg-9">{{ course.language }}<br/></span>
```
Comment (ismailsunni) [inline django_project/certification/templates/course/detail.html]: Course language. The other use sentence case. Just make sure to have the same case.

**Comment by sonlinux** [comment]
Thanks @ismailsunni

**Pull request merged**